docs/running-on-kubernetes.md

```diff
@@ -111,70 +162,96 @@ If our local proxy were listening on port 8001, we would have our submission loo
       --conf spark.app.name=spark-pi \
       --conf spark.kubernetes.driver.docker.image=kubespark/spark-driver:v2.1.0-kubernetes-0.1.0-alpha.2 \
       --conf spark.kubernetes.executor.docker.image=kubespark/spark-executor:v2.1.0-kubernetes-0.1.0-alpha.2 \
+      --conf spark.kubernetes.initcontainer.docker.image=kubespark/spark-init:v2.1.0-kubernetes-0.1.0-alpha.2 \
```

Comment:
I think a better question might be if the API for specifying Docker images can be in a different format than it is now. For example, a different API would be to have options for a Docker repository and a version for all of the images, and then Spark will just use the names `spark-driver`, `spark-init`, and `spark-executor` every time, only switching on the version and repository configuration. I think trying to infer the version of one thing given the versions of other things is too opaque.